index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <!-- Viewport? -->
```

Comment:
Your markup looks good.
I suggest that you add the meta viewport tag. This is a best practice that creates a consistent experience for a wider range of browsers and viewport sizes.